src/index.js

```diff
@@ -1,5 +1,7 @@
 import { app, BrowserWindow } from 'electron'
 
+require('electron-reload')(__dirname)
```

Comment:
This will probably work. But we could make things slightly more efficient with something like: 
```
require('electron-reload')(`${__dirname}/**/*.{css,js}`)
```

Actually this might be a generally good idea as we are constantly writing `.json` files into the `people` directory inside of `src`. God only knows what the app'll do if it tries to reload every time we try and write one of these.